astropy/coordinates/tests/test_frames.py

```diff
@@ -1437,6 +1437,25 @@ def test_galactocentric_defaults(reset_galactocentric_defaults):
         else:
             assert getattr(galcen_40, k) == getattr(galcen_latest, k)
 
+    # test validate Galactocentric
+    with galactocentric_frame_defaults.set('latest'):
+        params = galactocentric_frame_defaults.validate(galcen_latest)
+        references = galcen_latest.frame_attribute_references
+
+        assert galactocentric_frame_defaults._value == params
+        assert galactocentric_frame_defaults._references == references
+        assert galactocentric_frame_defaults._state == dict()
+
+    # Test not one of accepted parameter types
+    with pytest.raises(ValueError):
+        galactocentric_frame_defaults.validate(ValueError)
+
+    # test value property
```

Comment:
With that, I think one can remove this - private properties should be implementation details, ideally.

Reply:
I was mostly doing it for code coverage. I don't think it was ever hit in a test.